data/fwupd.service.in

```diff
@@ -18,6 +18,7 @@ MemoryDenyWriteExecute=yes
 NoNewPrivileges=no
 PrivateDevices=no
 PrivateTmp=true
+ProcSubset=pid
```

Comment:
Does this allow us to read `/proc/1/cgroup`?

Reply:
I am not 100% certain. According to documentation:

subset=pid hides all top level files and directories in the procfs that are not related to tasks.

Source: https://docs.kernel.org/filesystems/proc.html#mount-options

Is there a way I can test this?

Reply:
After some further digging, it is also defined (default setting) as:

Service has full access to non-process /proc files (/proc subset=)

Reply:
I think this means it's okay?

	sudo systemd-run  --wait -p ProcSubset=pid --pty --collect --service-type=exec cat /proc//1/cgroup
	Running as unit: run-u3717.service
	Press ^] three times within 1s to disconnect TTY.
	0::/init.scope
	Finished with result: success

Reply:
Looks readable to me with that output yes. ProcSubset is the setting I was most uncertain about in this PR.